cpp/utils.cpp

```diff
@@ -14,29 +14,34 @@ namespace opsqlite {
 namespace jsi = facebook::jsi;
 
 jsi::Value toJSI(jsi::Runtime &rt, const JSVariant &value) {
-  if (std::holds_alternative<bool>(value)) {
-    return std::get<bool>(value);
-  } else if (std::holds_alternative<int>(value)) {
-    return jsi::Value(std::get<int>(value));
-  } else if (std::holds_alternative<long long>(value)) {
-    return jsi::Value(static_cast<double>(std::get<long long>(value)));
-  } else if (std::holds_alternative<double>(value)) {
-    return jsi::Value(std::get<double>(value));
-  } else if (std::holds_alternative<std::string>(value)) {
-    auto str = std::get<std::string>(value);
-    return jsi::String::createFromUtf8(rt, str);
-  } else if (std::holds_alternative<ArrayBuffer>(value)) {
-    auto jsBuffer = std::get<ArrayBuffer>(value);
-    jsi::Function array_buffer_ctor =
-        rt.global().getPropertyAsFunction(rt, "ArrayBuffer");
-    jsi::Object o = array_buffer_ctor.callAsConstructor(rt, (int)jsBuffer.size)
-                        .getObject(rt);
-    jsi::ArrayBuffer buf = o.getArrayBuffer(rt);
-    memcpy(buf.data(rt), jsBuffer.data.get(), jsBuffer.size);
-    return o;
-  }
-
-  return jsi::Value::null();
+  return std::visit(
```

Comment:
This change is not *needed* but in theory it is faster according to chatGPT. I still need time to understand what does std::visit does and how std::decay_t is faster than a bunch of ifs accessing std::holds_alternative